index.html

```diff
@@ -0,0 +1,363 @@
+          </article>
+          <article>
+            <p class="fontawesome=icon">
+            <i class="fas fa-sync-alt"></i>
+            </p>
+            <h3 class="skills-item-description">REST APIs</h3>
+          </article>
+          <article>
+            <p class="fontawesome=icon">
+            <i class="fas fa-dumbbell"></i>
+            </p>
+            <h3 class="skills-item-description">Strong Work Ethic</h3>
+          </article>
+          <article>
+            <p class="fontawesome=icon">
+            <i class="far fa-smile-wink"></i>
+            </p>
+            <h3 class="skills-item-description">Great Attitude</h3>
+          </article>
+        </div>
+      </div>
+    </section>
+    <aside>
+      <div class="content-wrapper">
+        <div class="form-description">
+          <h2>wanna work with me?</h2>
+          <p>Let's do it! I'm available for freelance to create a beautiful and fully functional website, tailored to you.</p>
+          <input type="button" value="shoot me an email!" onclick="window.location.href='mailto:morganheuer@gmail.com'">
+        </div>
+      </div>
+    </aside>
+    <section id="portfolio">
+      <h2 class="title">Portfolio</h2>
+      <div class="content-wrapper">
+        <div id="portfolio-wrapper">
+          <div class="card">
+            <a href="https://github.com/morganheuer/task-list-api" target="_blank">
+              <img src="assets/tasks-1.png" alt="wordgame">
+            </a>
+            <h4>Task List API</h4>
+            <p>A web API to organize tasks. This API allows the client to create, read, update, and delete tasks. Other interesting features include: sort tasks, mark tasks complete, get feedback about our task list through Slack messages, organize tasks with goals.</p>
+            <br>
+            <p><strong>Built with: </strong>Python, Flask, SQLAlchemy, Postgres, Heroku, Postman</p>
+            <br>
+            <a href="https://secret-cliffs-20268.herokuapp.com/goals/1/tasks" target="_blank"><p>Launched via heroku here!</p></a>
+          </div>
+          <div class="card">
+            <a href="https://morganheuer.github.io/guess-the-word/" target="_blank">
+              <img src="assets/wordgame.png" alt="wordgame">
+            </a>
+            <h4>Guess the Word</h4>
+            <p>The game starts with a word on the screen. Circle symbols act as placeholders for each letter. Players start by entering a letter. If they guess correctly, the letter appears in the word. If they guess incorrectly, they learn how many guesses they have remaining. The player can also see which letters they've already guessed. The game lets the player know if they've already guessed a letter or entered a non-alphabetic character!</p>
+            <br>
+            <p><strong>Built with: </strong>HTML, CSS, JavaScript</p>
+            <br>
+            <a href="https://github.com/morganheuer/guess-the-word" target="_blank"><p>Check out the code files here!</p></a>
```

Comment:
Try to avoid using `<br>` tags in this way. It's more flexible to use CSS to control the spacing between these elements.